features/support/send_sigint_during_bisect.rb

```diff
@@ -5,8 +5,8 @@ module RSpec::Core::Formatters
   BisectProgressFormatter = Class.new(remove_const :BisectProgressFormatter) do
     RSpec::Core::Formatters.register self
 
-    def bisect_round_finished(notification)
-      return super unless notification.round == 3
+    def bisect_round_started(notification)
```

Comment:
I've renamed these to give them all the same prefix:

* `bisect_round_started`
* `bisect_round_ignoring_ids`
* `bisect_round_detected_multiple_culprits`

There's still the split for the different exit conditions, but at least now they're all more obviously a set.